lib/index.js

```diff
@@ -102,6 +125,25 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 
   app.options(accountPath, cors());
 
+  app.get(
+    `${accountPath}/caps`,
+    ensureAuthenticated,
+    cors(),
+    asyncHandler(async (req, res) => {
+      /**
+       * @func getCapabilitiesForAnAccount
+       * @param {Request} req
+       * @param {Response} res
+       * @description uses the account param to get
+       * all the capabilities for an account
+       * @route /:account/caps
+       */
+      const {account: id} = req.params;
+      const identities = [];
```

Comment:
Is there a reason for this declaration?  If it's not being used, then maybe we don't need it?

Reply:
basically getCaps requires an array (it has no default value for identities). So passing in an empty one means we get all capabilities. I think maybe @dlongley shoulds weigh in here on if this api should take in an identities array via json so we can restrict identity searches right now we just get all. so basically it did was it intended to get the reviewers to discuss this.

Reply:
We're trying to deprecate all of the `identities` stuff (it's not clear if we will be able to yet), so let's not include any of that here in new code.

Reply:
can you clarify because I believe the getCapabilities method requires an identity array right now. I will check on that.

Reply:
`identities` is optional: https://github.com/digitalbazaar/bedrock-account/blob/master/lib/index.js#L484-L496